schema/mysql/v57/temporal/schema.sql

```diff
@@ -259,7 +259,7 @@ CREATE TABLE cluster_membership
 (
     membership_partition INT NOT NULL,
     host_id              BINARY(16) NOT NULL,
-    rpc_address          VARCHAR(15) NOT NULL,
+    rpc_address          VARCHAR(128) NOT NULL,
```

Comment:
IPv6: `FEDC:BA98:7654:3210:FEDC:BA98:7654:3210` (39 characters). Why 128?

Reply:
`:` is for human only?

Reply:
just realized, ipv6 is 128 bits, not bytes?

Reply:
My thoughts were:
* trying to keep VARCHAR limit as low as possible doesn't provide any performance or storage benefits;
* a relaxed limit would allow storing not only IP addresses but FQDNs, which may save a database migration in the future :)
So I chose 128 as a power of 2 large enough to accommodate most `fqdn:port` RPC addresses.

Reply:
cassandra schema is still using `inet`, which limits the ipv4 / ipv6
https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cql_data_types_c.html

Reply:
`inet` can be converted to `blob` (according to [CQL type compatibility table](https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cql_data_types_c.html#cql_data_types_c__cql_data_type_compatibility)), should the need ever emerge.

I chose increased an VARCHAR length purely from forward-compatibility considerations. If you think this isn't necessary, I would suggest lowering it to 45 (https://stackoverflow.com/a/166157/1577116).